indexer/packages/postgres/src/stores/affiliate-info-table.ts

```diff
@@ -94,6 +97,165 @@ export async function findById(
     .returning('*');
 }
 
+export async function updateInfo(
```

Comment:
can you add a jsdoc explaining what this function does? Also I know you merged the logic to update persistentcache in this function just like in updateWalletVolume, but it might be better to separate it. When I read updateInfo, I don't expect that the persistent cache is being updated.